spec/template.dd

```diff
@@ -360,14 +360,15 @@ $(GNAME TemplateThisParameter):
         infer the mutability of the `this` reference. For example, if
         `this` is `const`, then the function is marked `const`.)
 
+        $(SPEC_RUNNABLE_EXAMPLE_COMPILE
         ---
         import std.stdio;
```

Comment:
This import is now not necessary anymore

Reply:
Good point, removed.